client/src/components/RunPipelineMenu.vue

```diff
@@ -0,0 +1,136 @@
+<script>
+import { runPipeline } from '@/common/viame.service';
+import { mapActions } from 'vuex';
+
+export default {
+  props: {
+    selected: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  computed: {
+    selectedEligibleClips() {
+      return this.selected.filter(
+        ({ _modelType, meta }) => _modelType === 'folder' && meta && meta.annotate,
+      );
+    },
+    pipelinesNotRunnable() {
+      return this.selectedEligibleClips.length < 1 || !this.pipelines;
+    },
+  },
+
+  asyncComputed: {
+    pipelines() {
+      return this.fetchPipelines();
+    },
+  },
+
+  created() {
+    this.fetchPipelines();
+  },
```

Comment:
Here you're calling `this.fetchPipelines()` twice, but you're only relying on the one called in the `pipelines` computed property. IMO you should map state here.